Fix name of the branch in template_workspace.yaml for the `fortress` installation

## Environment
* OS Version: <!-- e.g, Ubuntu 22.04 -->
* Source or binary build?
   * Branch: [`fortress`](https://github.com/gazebosim/ros_gz_project_template/tree/fortress)
   * Commit: https://github.com/gazebosim/ros_gz_project_template/commit/ffcd56186308630c0afd69900800b4f7507937d7

## Description
When trying to build from source the repository from the `fortress` branch I experienced a CMake error.

## Steps to reproduce
Follow the [Usage](https://github.com/gazebosim/ros_gz_project_template/tree/fortress?tab=readme-ov-file#usage) section of the README at the commit.

## Output
Following error:
~~~
--- stderr: ros_gz_example_gazebo                         
CMake Error at CMakeLists.txt:13 (find_package):
  By not providing "Findgz-cmake3.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "gz-cmake3", but CMake did not find one.

  Could not find a package configuration file provided by "gz-cmake3" with
  any of the following names:

    gz-cmake3Config.cmake
    gz-cmake3-config.cmake

  Add the installation prefix of "gz-cmake3" to CMAKE_PREFIX_PATH or set
  "gz-cmake3_DIR" to a directory containing one of the above files.  If
  "gz-cmake3" provides a separate development package or SDK, be sure it has
  been installed.
~~~

--- a/template_workspace.yaml
+++ b/template_workspace.yaml
@@ -3,4 +3,4 @@ repositories:
   ros_gz_project_template:
     type: git
     url: https://github.com/gazebosim/ros_gz_project_template.git
-    version: main
+    version: fortress

